packages/hardhat-core/sample-project/hardhat.config.js

```diff
@@ -3,7 +3,7 @@ require("@nomiclabs/hardhat-waffle");
 // This is a sample Hardhat task. To learn how to create your own go to
 // https://hardhat.org/guides/create-task.html
 task("accounts", "Prints the list of accounts", async () => {
-  const accounts = await ethers.getSigners();
+  const accounts = await hre.ethers.getSigners();
```

Comment:
What's the reason for this change? Normally people who prefer to use the global `hre` use its fields.

More context: We didn't define `global.hre` initially. We added it because it made giving support easier, as we know that `hre` will always be available for people who use the global stuff, and clear for people importing it explicitly.

Reply:
I brought this up on a call with @fvictorio . I complained that it felt weird to me that we'd use `ethers` unqualified like that (with no import), and if I remember correctly he said that it was an oversight and that we should be using `hre.`.

Reply:
@alcuadrado notice that this is within a task in the hardhat config. It seems very weird to me that the global variables from the `hre` are available there (although I understand why that happens).

That being said, I just now noticed that we should also add the hre to parameters:

```
task("accounts", "Prints the list of accounts", async (taskArgs, hre) => {
```

Reply:
I added the parameters in https://github.com/nomiclabs/hardhat/pull/1554/commits/5f11182769677497a57031b790baea1939fc7bf9

Reply:
Looks much better with the params!

What I found it confusing is that it was using the `hre` global variable, instead of directly going for `ethers`.